include/swift/AST/IRGenOptions.h

```diff
@@ -239,6 +239,9 @@ class IRGenOptions {
   /// The DWARF version of debug info.
   uint8_t DWARFVersion = 4;
 
+  /// Enables DWARF Fission and sets the file path for .dwo output.
+  std::string SplitDwarfOutput;
```

Comment:
Perhaps `DWOPath` or `DWOOutput` is more appropriate of a name?

Reply:
Note Stefan's notes about the names [here](https://github.com/apple/swift/pull/70296#issuecomment-1857986159). I don't have preferences. Will update per suggestions.

Reply:
LLVM calls it SplitDebugFileName (https://llvm.org/doxygen/classllvm_1_1DICompileUnit.html#af664fcb825cb6ffc1567fc2bc244173f). Personally, as long as we have a "Path" or "FileName" suffix I'm fine with either DWO or Split as the prefix.

Reply:
Done